README.md

```diff
@@ -5,6 +5,7 @@
 [![Code coverage][codecov-badge]][codecov-link]
 [![BSD-2-Clause][license-badge]][license-link]
 [![Documentation][doc-badge]][doc-link]
+[![User manual][zenodo-badge]][zenodo-link]
```

Comment:
Thanks. My only question here is whether we want to prominently point people to the PDF version of the website docs. Placed in this way, I'd think it's different content. Maybe the documentation site and/or later in the README should just say that equivalent information is available in PDF form at [zenodo link].

Reply:
I was wondering that too. I had it as a badge down with the citations  section and it looked out of place. Maybe a simple link would look less so

Reply:
Yeah, I think over there with or without badge or clarifying text saying that it's a PDF equivalent of the documentation website.

Reply:
Updated